JWBestPracticeApps/Podfile

```diff
@@ -1,30 +1,30 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '8.0'
+platform :ios, '11.0'
 # Uncomment this line if you're using Swift
 # use_frameworks!
 
 target 'JWBestPracticeApps' do
 
-pod 'JWPlayer-SDK', '~> 2.0'
+pod 'JWPlayer-SDK', '~> 3.2'
```

Comment:
why not 3.0 ? Doesn't  ~> 3.2 limit the versions to 3.2.x ? Ideally we want 3.x

Reply:
Done.

Reply:
@karimMourra I am a bit surprised, though, since you were the one that taught me this, when you explained why we specify the Chromecast framework as far as the build number; namely, that there is a tradeoff between flexibility and stability. We want to give flexibility to those we support downstream (so we support iOS 9.0, not 9.3), and we require stability from our upstream dependencies (so we specify Google Chromecast 2.10.0.4070).

I'm happy to change it to 3.0, but there are things that will break (for example, JWPlayerError.h was not exposed, and then not in the umbrella header). In an app demo for prospective or new customers looking for guidance for their new integration, I figure it's reasonable to assume that they will be using the most recent version of the SDK, barring some compelling reason for them to downgrade. What did I miss?

Reply:
The podfile should always download the latest jwplayer sdk. The Chromecast version will not change as often, therefore it should be fixed. Also we test against a specific chromecast version. 
You can't compare OS versions to SDK versions.